src/AdminConsole/Pages/Account/Login.cshtml.cs

```diff
@@ -32,16 +39,34 @@ public async Task<IActionResult> OnPost(string email, string? returnUrl = null)
     {
         returnUrl = Url.Page("/Organization/Overview");
 
+        var user = await _signInManager.UserManager.FindByEmailAsync(email);
+
+        if (user == null)
+        {
+            return RedirectToPage();
```

Comment:
This allows for user enumeration.

When we don't find a user currently, we still show that an email has been sent to the address provided.

If we want to protect against that, we should probably respond similarly to a successful submission.